.github/workflows/github-actions-pipeline.yml

```diff
@@ -0,0 +1,92 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'stacks/**'
```

Comment:
I wouldn't exclude the other paths, we'd still want to run tests etc. if anything changes.